docs/release-notes/1.0.0/release-1.0.0.md

```diff
@@ -0,0 +1,98 @@
+* Expose `Microsoft.Spark.Sql.Streaming.StreamingQueryManager` ([#670](https://github.com/dotnet/spark/pull/670))([#690](https://github.com/dotnet/spark/pull/690))
+* Update to `Microsoft.Dotnet.Interactive` 1.0.0-beta.20480.3 in `Microsoft.Spark.Extensions.DotNet.Interactive` ([#694](https://github.com/dotnet/spark/pull/694))
+* Broadcast encryption support ([#489](https://github.com/dotnet/spark/pull/489))
+
+### Bug Fixes
+
+* Fix concurrent reading of broadcast variable file during deserialization ([612](https://github.com/dotnet/spark/pull/612))
+* Return UTF-8 encoded string from JVM => CLR ([661](https://github.com/dotnet/spark/pull/661))
+* Add JVM CallbackClient connection back to connection pool ([681](https://github.com/dotnet/spark/pull/681))
+* JvmBridge/Netty blocking connection deadlock mitigation ([714](https://github.com/dotnet/spark/pull/714))
+
+### Infrastructure / Documentation / Etc.
+
+* Update Delta Lake tests against Delta Lake 0.6.1 ([#588](https://github.com/dotnet/spark/pull/588))
+* Add more `DataFrame` examples ([#599](https://github.com/dotnet/spark/pull/599))
+* Update ubuntu instructions by prepending current directory in sample command ([#603](https://github.com/dotnet/spark/pull/603))
+* [Delta Lake](https://github.com/delta-io/delta) version annotations to `Microsoft.Spark.Extensions.Delta` ([#632](https://github.com/dotnet/spark/pull/632))
+* [Hyperspace](https://github.com/microsoft/hyperspace) version annotations to `Microsoft.Spark.Extensions.Hyperspace` ([#696](https://github.com/dotnet/spark/pull/696))
+* Refactor unit tests ([#638](https://github.com/dotnet/spark/pull/638))
+* Add `SimpleWorkerTests` ([#644](https://github.com/dotnet/spark/pull/644))
+* Update to `Arrow` 0.15.1 ([#653](https://github.com/dotnet/spark/pull/653))
+* Remove `Microsoft.Spark.Extensions.Azure.Synapse.Analytics` from repo ([#687](https://github.com/dotnet/spark/pull/687))
+* Move `Microsoft.Spark.Experimental` to `Microsoft.Spark` ([#691](https://github.com/dotnet/spark/pull/691))
+* Fix `DaemonWorkerTests.TestsDaemonWorkerTaskRunners` and `CallbackTests.TestCallbackServer` tests ([#699](https://github.com/dotnet/spark/pull/699))
+* Improve build pipeline ([#667](https://github.com/dotnet/spark/pull/667))([#692](https://github.com/dotnet/spark/pull/692))([#697](https://github.com/dotnet/spark/pull/697))([#705](https://github.com/dotnet/spark/pull/705))([#717](https://github.com/dotnet/spark/pull/717))([#719](https://github.com/dotnet/spark/pull/719))([#720](https://github.com/dotnet/spark/pull/720))
+
+
+### Breaking Changes
+
+* No backward/forward compatibility between `Microsoft.Spark` and `Microsoft.Spark.Worker` if `version` < 1.0.0
+
+### Known Issues
+
+* Broadcast variables do not work with [dotnet-interactive](https://github.com/dotnet/interactive) ([#561](https://github.com/dotnet/spark/pull/561))
```

Comment:
Do we also want to mention [this issue](https://github.com/dotnet/spark/issues/619) here since it is related?